### PR TITLE
feat(n8n): add docs about n8n

### PR DESCRIPTION
**Pull request by edumaciel10**

### CODE REVIEW

**Review by sibelius — APPROVED** [review]

**Review by daniloab — APPROVED** [review]

**Review by noghartt — APPROVED** [review]

**Review by fersilva16 — APPROVED** [review]
Add prints

File: docs/integrations/woovi-n8n.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+4. Pesquisar por `n8n-nodes-woovi`
+
+## O que é possível fazer com o nó da Woovi no N8N?
+
+Com o nó da Woovi você consegue:
+
+- Criar cobranças
+- Buscar por cobranças [BETA]
+- Reembolsar cobranças [BETA]
+- Realizar saques [BETA]
```
[the remark this answers is not in the document]
Comment (edumaciel10) [reply docs/integrations/woovi-n8n.md:31]: we need beta to create marketing for the customers

[the remark this answers is not in the document]
Comment (daniloab) [reply docs/integrations/woovi-n8n.md:31]: prefer remove beta

[the remark this answers is not in the document]
Comment (sibelius) [reply docs/integrations/woovi-n8n.md:31]: (Request Access)

File: docs/integrations/woovi-n8n.md [hunk 2]
```diff
@@ -26,11 +25,13 @@ Para encontrar o nó da Woovi no N8N, você deve:
 Com o nó da Woovi você consegue:
 
 - Criar cobranças
-- Buscar por cobranças [BETA]
-- Reembolsar cobranças [BETA]
-- Realizar saques [BETA]
+- Buscar por cobranças (Request Access)
```
Comment (sibelius) [inline docs/integrations/woovi-n8n.md:28]: Solicite acesso

**Pull request merged**